learning/katas/go/Core Transforms/Combine/Combine PerKey/task.md

```diff
@@ -0,0 +1,47 @@
+<!--
+  ~  Licensed to the Apache Software Foundation (ASF) under one
+  ~  or more contributor license agreements.  See the NOTICE file
+  ~  distributed with this work for additional information
+  ~  regarding copyright ownership.  The ASF licenses this file
+  ~  to you under the Apache License, Version 2.0 (the
+  ~  "License"); you may not use this file except in compliance
+  ~  with the License.  You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~  Unless required by applicable law or agreed to in writing, software
+  ~  distributed under the License is distributed on an "AS IS" BASIS,
+  ~  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~  See the License for the specific language governing permissions and
+  ~  limitations under the License.
+  -->
+
+# Combine - Combine PerKey
+
+After creating a keyed PCollection (for example, by using a GroupByKey transform), a common pattern 
+is to combine the collection of values associated with each key into a single, merged value. This 
+pattern of a GroupByKey followed by merging the collection of values is equivalent to Combine 
+PerKey transform. The combine function you supply to Combine PerKey must be an associative 
+reduction function or a subclass of CombineFn.
```

Comment:
There are no "subclasses" in Go.

Reply:
Apologies, I missed that.  Would it be clear if we just remove "or a subclass of CombineFn" such that the full sentence reads:  "The combine function you supply to beam.CombinePerKey must be an associative reduction function"?